src/style.css

```diff
@@ -0,0 +1,76 @@
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir-Book.ttf);
+  font-style: normal;
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir-Heavy.ttf);
+  font-style: normal;
+  font-weight: 800;
+}
+
+.search-form {
+  position: relative;
+  display: flex;
+  margin-top: 20px;
+}
+
+.search-form__input {
+  width: 100%;
+  box-sizing: border-box;
+  border-radius: 4px;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px 0 #3d4e611a;
+  font-family: Avenir, sans-serif;
+  font-style: normal;
+  color: #3d4e61;
+}
+
+.search-form__input:hover {
+  box-shadow: 0 3px 8px 0 #3d4e6133;
+}
+
+.search-form__input:focus {
+  border-radius: 3px;
+  outline: none;
+  font-weight: 800;
+  background: linear-gradient(180deg, #fff, #f6f6f7);
+}
+
+.search-form__input--big {
+  height: 70px;
+  padding: 0 62px;
+  font-size: 16px;
+  line-height: 22px;
+}
+
+.search-form__icon--big {
+  position: absolute;
+  left: 26px;
+  bottom: 26px;
+  width: 19px;
+  height: 19px;
+}
+
+.search-form__input--big:focus {
+  box-shadow: 0 4px 4px 0 #00000040;
+  text-shadow: 0 4px 4px #00000040;
+}
+
+.search-form__input--small {
+  height: 42px;
+  padding: 0 33px;
+  font-size: 14px;
+  line-height: 20px;
+}
+
+.search-form__icon--small {
+  position: absolute;
+  left: 13px;
+  bottom: 16px;
```

Comment:
Center icons in a reusable way. Now it depends on the image size. You can use align-items for the parent block or top + transition transform properties

Reply:
Test failed, but issue resolved.